src/util/states.ts

```diff
@@ -529,11 +532,12 @@ export function blurSeriesFromHighlightPayload(
             el = data.getItemGraphicEl(current++);
         }
     }
+    const excludedSeriesIndexes = payload.seriesIndex || [];
```

Comment:
`seriesIndex` is one of the possible queries. Developers can also use `payload.seriesId` to query target series. Perhaps we need to query the highlight `seriesModel` list in `echarts.ts` and pass it to this function.

Also, I'm not sure if there is a logic issue:

If we have `focus: 'self'` in one series and dispatch a `highlight` action on this series. This series will also be in this `excludedSeriesIndexes` and not perform the `focus: 'self'` strategy.

Reply:
> Also, I'm not sure if there is a logic issue:
>
> If we have focus: 'self' in one series and dispatch a highlight action on this series. This series will also be in this excludedSeriesIndexes and not perform the focus: 'self' strategy.

Yes, There is a logic issue. I add another testcase for this.
Please take a look if this is the expected result~

![Jun-26-2021 20-31-19](https://user-images.githubusercontent.com/10973821/123513091-bd6afb00-d6bd-11eb-98ab-866431b0a98b.gif)